microsite/blog/2022-07-19-releasing-backstage-search-1.0.md

```diff
@@ -0,0 +1,63 @@
+---
+# prettier-ignore
+title: Releasing Backstage Search 1.0 
+author: Emma Indal, Spotify
+authorURL: https://www.linkedin.com/in/emma-indal
+---
+
+**TL;DR** If you’ve been waiting for Backstage Search to come out of beta, we’re excited to announce that [Backstage Search 1.0](https://backstage.io/docs/features/search/search-overview#backstage-search-10) is here!
+
+We first released the Backstage Search Platform over a year ago. Backstage Search Platform is a search experience built for you, by you.
+
+![Releasing Backstage Search 1.0](assets/22-07-19/releasing-backstage-search-v1.png)
+
+Backstage Search Platform makes it possible for you to search across your entire software ecosystem. With a Search platform that is customizable and extendable, you can identify your needs within your organization and set up search to solve these problems. Whether it is to search through the Software Catalog or technical documentation generated by the TechDocs plugin.
+
+<!--truncate-->
+
+## So what is Backstage Search 1.0?
+
+Since the first release of the Backstage Search Platform, we have seen a fantastic community grow around the search plugins. To mention just a few contributions – more search engine modules ([Elasticsearch](https://www.npmjs.com/package/@backstage/plugin-search-backend-module-elasticsearch), originally [contributed](https://github.com/backstage/backstage/pull/6569) by Roadie
```

Comment:
can we change the formatting of this section? this is the markdown i have for this section: 

### So what is Backstage Search 1.0?

Since the first release of the Backstage Search Platform, we have seen a fantastic community grow around the search plugins. To mention just a few of these contributions: 
* More search engine modules: [Elasticsearch](https://www.npmjs.com/package/@backstage/plugin-search-backend-module-elasticsearch) originally [contributed](https://github.com/backstage/backstage/pull/6569) by Roadie and [Postgres](https://www.npmjs.com/package/@backstage/plugin-search-backend-module-pg) originally [contributed](https://github.com/backstage/backstage/pull/7316) by SDA SE
* More components: Quick search modal, originally [contributed](https://github.com/backstage/backstage/pull/7316) by Expedia Group
* More plugin integrations: TechDocs also [contributed](https://github.com/backstage/backstage/pull/6375) by Roadie and [Software Catalog](https://github.com/backstage/backstage/pull/4515) 

These are just a few contributions of the countless enhancements and improvements made by contributors across the globe. With a well adopted and tested search platform we feel confident to release Backstage Search as 1.0. 


I'm sorry i don't know how to suggest code changes

Reply:
> Since the first release of the Backstage Search Platform, we have seen a fantastic community grow around the search plugins. To mention just a few of these contributions:
> 
> * More search engine modules: [Elasticsearch](https://www.npmjs.com/package/@backstage/plugin-search-backend-module-elasticsearch) originally [contributed](https://github.com/backstage/backstage/pull/6569) by Roadie and [Postgres](https://www.npmjs.com/package/@backstage/plugin-search-backend-module-pg) originally [contributed](https://github.com/backstage/backstage/pull/7316) by SDA SE
> * More components: Quick search modal, originally [contributed](https://github.com/backstage/backstage/pull/7316) by Expedia Group
> * More plugin integrations: TechDocs also [contributed](https://github.com/backstage/backstage/pull/6375) by Roadie and [Software Catalog](https://github.com/backstage/backstage/pull/4515)
> 
> These are just a few contributions of the countless enhancements and improvements made by contributors across the globe. With a well adopted and tested search platform we feel confident to release Backstage Search as 1.0.

Sure, I applied your suggestions, thanks again for looking into this 🙌🏻
And speaking of code suggestions, you might find this short tutorial interesting:
![Code suggestions](https://i.stack.imgur.com/Qu179.gif)